script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -482,18 +522,18 @@ Besides, aren't you the one who picked a fight with her? If you had played nice,
 [sha:becc3a0120bee383919867202ac16e224738caae]{
 -- Page 276, Offset 27816.
 -- わたしだって考えなしではありません、この一週間ずっと自重してきたんです！
-I don't act thoughtlessly. I've been cautious the whole week!
+It's not like I act without thinking. I've been holding myself back all week!
```

Comment:
I have been sounds like she still is, when she clearly failed with that the last day.
>I held myself back this whole week!